mautic/mautic
# Focus: Remove click event anywhere to close modal

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
If the page contains a consent window, then it's impossible to use a popup or notification.  
There is on click event that closes it after clicking anywhere. We removed it and now it's possible to close the modal just with the cross button. 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add consent window to the page to confirm cookie
3. Create notification and put it to the page
4. If you click on the consent cookie window, then the notification is closed too.
5. After PR you are able to close it just on cross button in notification
